### PR TITLE
[14.0][FIX] cetmix_tower_server: Clean up Python command execution result

### PR DESCRIPTION
**Pull request by dessanhemrayev**
Renamed method _parse_ssh_command_results to  _parse_command_results for using not only for SSH but for any command including Python one.

Task: 3952

### CODE REVIEW

File: cetmix_tower_server/tests/test_command.py [hunk 1]
```diff
@@ -837,3 +844,27 @@ def test_execute_python_code(self):
             command_result["error"],
             "Error in command result must be set to None",
         )
+
+    def test_execute_python_code_with_secret(self):
```
Comment (ivs-cetmix) [inline cetmix_tower_server/tests/test_command.py:869]: We need to check if secret key is replaced in the command output. I didn't manage to find such a flow in this test.

File: cetmix_tower_server/tests/test_command.py [hunk 2]
```diff
@@ -41,6 +41,19 @@ def setUp(self, *args, **kwargs):
     COMMAND_RESULT = {"exit_code": 0, "message": "New command was created"}
 else:
     COMMAND_RESULT = {"exit_code": -1, "message": "error"}
+    """,
+            }
+        )
+
+        self.command_create_new_command_2 = self.Command.create(
+            {
+                "name": "Create new command",
```
Comment (ivs-cetmix) [inline cetmix_tower_server/tests/test_command.py]: Command name as well as the name of the variable it's assigned don't look relevant. Which degraded the code readability.
There should be something like "Python command with secret" or similar.

**Review by ivs-cetmix — CHANGES_REQUESTED** [review]

File: cetmix_tower_server/tests/test_command.py (hunk 2 above)
Comment (dessanhemrayev) [reply cetmix_tower_server/tests/test_command.py]: Thanks, I'll fix it

File: cetmix_tower_server/tests/test_command.py (hunk 1 above)
Comment (dessanhemrayev) [reply cetmix_tower_server/tests/test_command.py:869]: Thanks, I'll fix it

File: cetmix_tower_server/models/cx_tower_key.py [hunk 3]
```diff
@@ -450,6 +450,9 @@ def _replace_with_spoiler(self, code, key_values):
 
         # Replace keys with values
         for key_value in key_values:
+            # If key_value includes quotes, remove them for the replacement
+            if key_value.startswith('"') and key_value.endswith('"'):
+                key_value = key_value[1:-1]
```
Comment (GabbasovDinar) [inline cetmix_tower_server/models/cx_tower_key.py]:
```suggestion
            key_value = key_value.strip('"')
```

**Review by GabbasovDinar — CHANGES_REQUESTED** [review]

**Pull request closed without merging**